jbmc/src/java_bytecode/java_bytecode_parse_tree.h

```diff
@@ -221,6 +221,7 @@ class java_bytecode_parse_treet
     bool is_static_class = false;
     bool is_anonymous_class = false;
     bool attribute_bootstrapmethods_read = false;
+    irep_idt outer_class;
```

Comment:
What happens if there is no outer class? Would it make sense to have an `optionalt` here?
Else maybe comment how absence of outer classes is handled (I guess it's an empty identifier then?)